### PR TITLE
[15.0][MIG] sale_order_line_price_history: Migration to 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT36522

Supersede https://github.com/OCA/sale-workflow/pull/1923:

- I kept the changes made by @bizzappdev and but I have done what i proposed here https://github.com/OCA/sale-workflow/pull/1923#pullrequestreview-1016981475 since @bizzappdev doesn't answer to my comment
- I added a line to roadmap readme section

### CODE REVIEW

**Comment by ernestotejeda** [comment]
I just added a commit as a workaround for one of the roadmap specifications, could you check it @chienandalu @pedrobaeza ?

File: sale_order_line_price_history/wizards/sale_order_line_price_history.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                </group>
+                <group string="Sale history">
+                    <field name="line_ids" nolabel="1" readonly="1">
+                        <form>
+                            <div
+                                class="oe_button_box"
+                                name="button_box"
+                                attrs="{'invisible': [('history_sale_order_line_id', '=', False)]}"
+                            >
+                                <button
```
Comment (pedrobaeza) [inline sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: Put this button as well in the tree view

Comment (chienandalu) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: This one belongs to the tree inside the sale form.

**Review by chienandalu — APPROVED** [review]

File: sale_order_line_price_history/wizards/sale_order_line_price_history.xml (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: I know, but that's why I want it as well in the tree view.

Comment (ernestotejeda) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: When the popup opens, the wizard record hasn't been created yet, so the button on those lines will be disabled (normal odoo behavior).

Comment (pedrobaeza) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: I'm not saying that. I just want to have the button "Set price" on the tree view of the wizard with an icon, only when the related sale line already exists.

Comment (ernestotejeda) [reply sale_order_line_price_history/wizards/sale_order_line_price_history.xml:24]: I wrote the limitation in the roadmap in the migration commit, please, check it out.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**